moveit_demo_nodes/run_moveit_cpp/launch/run_moveit_cpp.launch.py

```diff
@@ -0,0 +1,92 @@
+        return None
+
+
+def generate_launch_description():
+    # moveit_cpp.yaml is passed by filename for now since it's node specific
+    moveit_cpp_yaml_file_name = get_package_share_directory('run_moveit_cpp') + "/config/moveit_cpp.yaml"
+
+    # Component yaml files are grouped in separate namespaces
+    robot_description_config = load_file('moveit_resources', 'panda_description/urdf/panda.urdf')
+    robot_description = {'robot_description' : robot_description_config}
+
+    robot_description_semantic_config = load_file('moveit_resources', 'panda_moveit_config/config/panda.srdf')
+    robot_description_semantic = {'robot_description_semantic' : robot_description_semantic_config}
+
+    kinematics_yaml = load_yaml('moveit_resources', 'panda_moveit_config/config/kinematics.yaml')
+    robot_description_kinematics = { 'robot_description_kinematics' : kinematics_yaml }
+
+    controllers_yaml = load_yaml('run_moveit_cpp', 'config/controllers.yaml')
+    moveit_controllers = { 'moveit_simple_controller_manager' : controllers_yaml }
+
+    ompl_planning_pipeline_config = { 'ompl' : {
+        'planning_plugin' : 'ompl_interface/OMPLPlanner',
+        'request_adapters' : """default_planner_request_adapters/AddTimeOptimalParameterization default_planner_request_adapters/FixWorkspaceBounds default_planner_request_adapters/FixStartStateBounds default_planner_request_adapters/FixStartStateCollision default_planner_request_adapters/FixStartStatePathConstraints""" ,
+        'start_state_max_bounds_error' : 0.1 } }
+    ompl_planning_yaml = load_yaml('moveit_resources', 'panda_moveit_config/config/ompl_planning.yaml')
+    ompl_planning_pipeline_config['ompl'].update(ompl_planning_yaml)
+
+    # MoveItCpp demo executable
+    run_moveit_cpp_node = Node(node_name='run_moveit_cpp',
+                               package='run_moveit_cpp',
+                               # TODO(henningkayser): add debug argument
+                               # prefix='xterm -e gdb --args',
+                               node_executable='run_moveit_cpp',
+                               output='screen',
+                               parameters=[moveit_cpp_yaml_file_name,
+                                           robot_description,
+                                           robot_description_semantic,
+                                           kinematics_yaml,
+                                           ompl_planning_pipeline_config,
+                                           moveit_controllers])
+
+    # RViz
+    rviz_config_file = get_package_share_directory('run_moveit_cpp') + "/launch/run_moveit_cpp.rviz"
+    rviz_node = Node(package='rviz2',
+                     node_executable='rviz2',
+                     node_name='rviz2',
+                     output='log',
+                     arguments=['-d', rviz_config_file],
+                     parameters=[robot_description])
+
+    # RobotStatePublisher
```

Comment:
I just saw that this doesn't include a robot state publisher (which I don't think not adding it will effect the demo)

```python
    # RobotStatePublisher
    robot_state_publisher_node = Node(
                            package='robot_state_publisher',
                            node_executable='robot_state_publisher',
                            node_name='robot_state_publisher',
                            output='screen',
                            arguments=[os.path.join(get_package_share_directory('moveit_resources'), 'panda_description', 'urdf', 'panda.urdf')]
                            )
```

And later 

```python
return LaunchDescription([ static_tf, rviz_node, robot_state_publisher_node, run_moveit_cpp_node, fake_joint_driver_node ])
```

Reply:
I think you looked at an outdated file. It's only the comment that's wrong now